Makefile

```diff
@@ -77,14 +77,20 @@ venv:
 
 test:
 	@echo $(H1)Running tests$(HEADER_EXTRA)$(H1END)
-	$(VENV_BIN)/python -m pytest $(COV) ./httpie $(COV) ./tests --doctest-modules --verbose ./httpie ./tests
+	$(VENV_BIN)/python -m pytest $(COV) ./httpie $(COV) ./tests --doctest-modules --verbose ./httpie ./tests $(MISSING)
```

Comment:
Could you revert those changes and propose them in a specific PR if you think it is interesting to keep :pray: ?

Reply:
certainly

Reply:
Done